### ISSUE TITLE
Fix localizations

### ISSUE THREAD

**Issue by ConfusedVorlon** [abbeycode/UnzipKit#42]
1.81 generates a warning when submitting to iTunes (re Unrecognized Locale -)
It looks like this is fixed in a more recent commit.

Could you push 1.82?

thank you.

**Comment by abbeycode** [comment]
Thanks for pointing this out. I didn't realize the issues I fixed would affect anything material. I'll work on this when I can (hopefully today).

**Comment by ConfusedVorlon** [comment]
thank you.

**Comment by ConfusedVorlon** [comment]
It is just a warning from iTunes, not an error, but would be nice to get
rid of it.

Thank you

______________________________
Small phone.  Terse response.

**Comment by abbeycode** [comment]
I've bumped the version, and pushed an update to CocoaPods. Thanks again for pointing this out!

**Comment by tomjpsun** [comment]
Hi ! 1.8.3 still gets this warning. Could you re-open the issue again ?

**Comment by abbeycode** [comment]
@tomjpsun Could you please give me more info, since I haven't seen that error?

1. How are you building UnzipKit (CocoaPods, Carthage, Directly embedded source)?
2. At what time are you receiving the error (build time, analyze time, submission to App Store)?
3. What exact error are you getting? Screenshot and copy-paste of text, if possible, would be great

Thanks!

**Comment by ConfusedVorlon** [comment]
Hi,

I'm building with cocoapods for a swift app using frameworks.

This is the error I get (it comes as an email after I upload an app to iTunes Connect)

> Dear developer,
> 
> We have discovered one or more issues with your recent delivery for "Stmpr - Write your own gif!". Your delivery was successful, but you may wish to correct the following issues in your next delivery:
> 
> Unrecognized Locale -
> The directory located at ( 'Payload/STMPR.app/Frameworks/UnzipKit.framework/UnzipKitResources.bundle/base.lproj' ) has an unrecognized locale name. Refer to the Language and Locale Designations guide for more information on naming your language-specific directories.
> 
> After you’ve corrected the issues, you can use Xcode or Application Loader to upload a new binary to iTunes Connect.
> 
> Regards,
> 
> The App Store team
> 
> 

Comparing unzipkit with other pods that don't generate an error; the format for unzipkit is 

/UnzipKit.framework/UnzipKitResources.bundle/base.lproj'

wheras other frameworks have

/UnzipKit.framework/UnzipKitResources.bundle/en.lproj'

(and other languages - but not base)

I'm guessing something about putting this into the framework means that base isn't ok, and you need to provide explicit language folders.

There is a plist entry which describes the native localisation region as en; so I don't think there is any ambiguity as to the default language.

tl;dr; I think you you change base.lproj to en.lproj, then everything will be splendid!

**Comment by abbeycode** [comment]
There might be a better way to handle it without moving off of the base internationalization ([documentation](https://developer.apple.com/library/content/documentation/MacOSX/Conceptual/BPInternational/InternationalizingYourUserInterface/InternationalizingYourUserInterface.html#//apple_ref/doc/uid/10000171i-CH3-SW2)), but using it wasn't really getting me anything, since the library doesn't present a UI. I switched to a strictly `en` localization in a branch, and will merge once I've worked out other issues going on in `master`. I won't close this until it's made it into the next release.

**Comment by abbeycode** [comment]
I released this update in v1.8.4. Please write back if you still see the problem persisting.

**Comment by stevenp** [comment]
Unfortunately I'm still seeing this warning when uploading an archive on 1.8.4.

![screen shot 2017-02-06 at 4 45 44 pm](https://cloud.githubusercontent.com/assets/181589/22669770/c2040682-ec8b-11e6-9703-c846879185f9.png)

**Comment by abbeycode** [comment]
Not sure how the full fix didn't get committed. Should be good now, but will verify.

**Comment by abbeycode** [comment]
The 1.8.5 release is pending a successful Travis CI build, but I did verify in the demo project that _only_ the `en.lproj` folder was in the UnzipKitResources bundle. I think this was the result of a bad merge. Somehow, the commit history was showing the file getting moved to the en.lproj directory as the last change, but it was still in base.lproj. I'll send an update once the new version is out on CocoaPods.

**Comment by abbeycode** [comment]
This should really be fixed this time. Let me know!
